Source/WebCore/platform/graphics/FontCascade.cpp

```diff
@@ -880,14 +836,22 @@ FontCascade::CodePath FontCascade::characterRangeCodePath(std::span<const UChar>
             continue;
         }
 
-        if (c < 0xFE00) // U+FE00 through U+FE0F Unicode variation selectors
-            continue;
-        if (c <= 0xFE0F)
-            return CodePath::Complex;
+        // Search for complex cases
+        auto* boundingCharacter = WTF::approximateBinarySearch<const UChar, const UChar>(complexCodePathRanges, complexCodePathRangesSize, c, keyExtractorUChar);
```

Comment:
Would a map be faster here ? We have `SortedArrayMap`

Reply:
Two different PRs got merged by accident.

Reply:
I was gonna say I didn't approve all this!